### PR TITLE
Returns keypad to its former stylistic glory

### LINKED ISSUE #3153: Keypad so ugly
image

The .simple-button style got removed a while back without considering the unintended consequences.

### PR DESCRIPTION
**Pull request by rtibbles**
Fixes #3153.

Summary of changes:
* Refactor software keyboard to use handlebars template
* Restyle software keyboard to properly leverage Bootstrap styling

image

### CODE REVIEW

File: kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<button class='simple-button orange' id='show-keyboard'> {{_ "Hide Keypad" }}</button>
+
+<div class='container-fluid' id='software-keyboard'>
+    {{#each keys }}
+        <div class="row">
+            <div class="col-xs-12">
```
Comment (MCGallaspy) [inline kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars]: Why not put each button in a `col-xs-4` container?

File: kalite/distributed/static/css/distributed/khan-lite.css [hunk 2]
```diff
@@ -820,50 +795,24 @@ a.disabled {
 }
 
 #software-keyboard div {
```
Comment (MCGallaspy) [inline kalite/distributed/static/css/distributed/khan-lite.css]: Could it be `#software-keyboard .row {`? Small change, I know, but more semantic.

File: kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars (hunk 1 above)
Comment (rtibbles) [inline kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars]: Tried it. Got this:
image

Comment (rtibbles) [inline kalite/distributed/hbtemplates/exercise/software-keyboard.handlebars]: Bootstrap buttons are designed to be displayed inline, so it makes sense to put them in the same div container.

**Comment by rtibbles** [comment]
@MCGallaspy I think this is the right fix as it stands.

**Comment by rtibbles** [comment]
Now fixes #3174 too.

**Comment by rtibbles** [comment]
image

Added ( and ) keys too.

**Comment by rtibbles** [comment]
N.B. - used character for space rather than 'Space' to avoid adding additional strings before release @aronasorman. @jtamiace if you can think of better ways to label it going forward, I am all ears, but no strings for now!

**Comment by MCGallaspy** [comment]
@rtibbles you don't think a key with whitespace will be unambiguous enough? :P
Looks good to me. Does @jtamiace have merging privileges?

**Comment by MCGallaspy** [comment]
Though I do wonder if that character will display properly on all systems/fonts that could potentially be used. Is that a valid concern? Should it be an image instead?

**Comment by rtibbles** [comment]
It's Unicode, so I think it will be OK - but I am not wedded to it, if you and @jtamiace think just a blank box is a better indicator, happy to go with that.

**Comment by MCGallaspy** [comment]
@rtibbles no, I agree that having something is better than just a blank button. (Much like doge speak, :P should be taken to mean the preceding comments isn't particularly serious.) If you don't anticipate any display issues I'm happy with this.

**Comment by rtibbles** [comment]
Sure, let's merge.

**Pull request merged**